RadixWallet/Clients/DeviceFactorSourceClient/DeviceFactorSourceClient+Live.swift

```diff
@@ -11,7 +11,9 @@ extension DeviceFactorSourceClient: DependencyKey {
 		@Dependency(\.personasClient) var personasClient
 		@Dependency(\.factorSourcesClient) var factorSourcesClient
 
-		let entitiesControlledByFactorSource: GetEntitiesControlledByFactorSource = { factorSource, maybeSnapshot in
+		let entitiesControlledByFactorSource: GetEntitiesControlledByFactorSource = {
+			factorSource,
+				maybeSnapshot in
```

Comment:
Why the crazy newlines?

Reply:
indeed crazy, reverting. CTRL M went crazy.. thx XCode